examples/resources/genesyscloud_knowledge_category/resource.tf

```diff
@@ -0,0 +1,9 @@
+resource "genesyscloud_knowledge_category" "example_category" {
+  knowledge_base_id = genesyscloud_knowledge.example_knowledgebase.id
```

Comment:
genesyscloud_knowledge -> genesyscloud_knowledge_knowledgebase

Reply:
thanks for catching this